bot/cogs/moderation/utils.py

```diff
@@ -28,6 +28,18 @@
 UserSnowflake = t.Union[UserObject, discord.Object]
 Infraction = t.Dict[str, t.Union[str, int, bool]]
 
+APPEAL_EMAIL = "appeals@pythondiscord.com"
+
+INFRACTION_TITLE = f"Please review our rules over at {RULES_URL}"
+INFRACTION_APPEAL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_AUTHOR_NAME = "Infraction information"
+
+INFRACTION_DESCRIPTION_TEMPLATE = (
+    "\n**Type:** {type}\n"
+    "**Expires:** {expires}\n"
+    "**Reason:** {reason}\n"
+)
```

Comment:
The newlines will be replaced by a single empty space when passed through `textwrap.shorten`:

![image](https://user-images.githubusercontent.com/44734341/86572307-0ab50080-bf73-11ea-882c-2306dab4dac8.png)

I think it's confusing to have the newlines here in the template if we know they won't be shown on Discord.

Reply:
I think then we should shorten reason only. How much this should display of reason? 1000 characters?

Reply:
Sounds like a good idea, although this would ideally be answered by someone who knows more about the character limits in embeds, afaik there's a limit on the total and a limit on the description too? Fwiw this isn't a regression, there's no newlines on the master branch either.

Reply:
The [Discord.js guide](https://discordjs.guide/popular-topics/embeds.html#embed-limits) has some clear documentation on embed limits

Reply:
Is this better now?